.env.template

```diff
@@ -0,0 +1,5 @@
+FLASK_PORT=5000
+FLASK_DEBUG=false
+FLASK_HOST=localhost
+FLASK_SECRET_KEY=ThisIsADevelopmentKey
+DB_NAME=afpy.db
```

Comment:
Please end your files by newlines, it's more `cat` friendly (and we all love cats).